tools/examples_with_valgrind.py

```diff
@@ -0,0 +1,183 @@
+import os
+import subprocess
+import time
+from collections import defaultdict
+import shlex
+
+example_args = {
+        "client_encryption":"opc.tcp://localhost:4840 client_cert.der client_key.der server_cert.der",
```

Comment:
you have added a description for all examples.
It would be more convenient to only describe the examples that need command-line arguments.
All other examples can be used without additional arguments by default.

That would be better because we have less breakage when a new example is added.

Reply:
There is no need to add any description for an example.
Only if it needs a client/server or arguments. 
I am using the get() method of the dictionary. If the key is not present, this returns None and since None evaluates to False in a Boolean context, no additional arguments or a client/server will be added.